src/Microsoft.Azure.Relay/HybridConnectionListener.cs

```diff
@@ -214,6 +216,16 @@ private set
         [EditorBrowsable(EditorBrowsableState.Never)]
         public bool UseBuiltInClientWebSocket { get; set; }
 
+        /// <summary>
+        /// Websocket's keep-alive interval.
+        /// </summary>
+        public TimeSpan KeepAliveInterval { get; set; }
+
+        /// <summary>
+        /// Custom ClientWebSocket Implementation.
+        /// </summary>
+        public IClientWebSocketFactory CustomClientWebSocketFactory { get; set; }
```

Comment:
>CustomClientWebSocketFactory [](start = 39, length = 28)

If `[EditorBrowsable(EditorBrowsableState.Advanced)]` goes in the other place, let's also do it here.

Reply:
added